docs/tools/mbed_cli_2/use.md

```diff
@@ -63,6 +63,22 @@ mbed-tools clone <example> <PATH>
 - [mbed-os-example-nfc](https://github.com/ARMmbed/mbed-os-example-nfc)
 - [mbed-os-example-sockets](https://github.com/ARMmbed/mbed-os-example-sockets)
 
+### Use a specific Mbed program library
+
+A Mbed program can use library dependencies at the revision specified in the ".lib" files.
+This ensures all dependencies are resolved and the versions are synchronised to the version specified in the library reference.
+
+    ```
+    mbedtools deploy <PATH>
```

Comment:
Do we usually use `mbed-tools` or `mbedtools` in this document? Make everything consistent, without changing too many of them.